config.ddev-drupal-core-dev.yaml

```
@@ -0,0 +1,6 @@
+# #ddev-generated
+# This file is placed by the justafish/ddev-drupal-core-dev addon.
+
+ddev_version_constraint: '>=v1.23.0-alpha1'
```

Might as well be v1.23.0 now.

```suggestion
ddev_version_constraint: '>=v1.23.0'
```